doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -40,11 +40,43 @@
     "SMOKE_TEST = True"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Uncomment and run the following line in order to install all the necessary dependencies:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "# !pip install numpy datasets \"transformers>=4.19.1\" \"pytorch_lightning>=1.6.5\""
+   ]
```

Comment:
Any particular reason that this needs to be commented out?

Reply:
nope, just followed what other examples did. 😂 I'll uncomment this